src/Files.App/Views/Properties/CustomizationPage.xaml

```diff
@@ -31,7 +32,7 @@
 			BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
 			BorderThickness="1"
 			ColumnSpacing="8"
-			CornerRadius="4"
+			CornerRadius="8"
```

Comment:
It could be it changed but the guidelines used to be to use `4px`.

Reply:
According to the MS Docs of the Mica Material, they use 8px for cards. But I’m not sure we should follow also in this case (cards inside 8px rounded frame)

Reply:
Let's try with 8px and see how it goes.